docs/guide/plugin-development.md

```diff
@@ -79,12 +79,12 @@ interface HookFilter {
    * @example
    * Include all `id`s that contain `node_modules` in the path.
    * ```js
-   * { id: 'node_modules' }
+   * { id: '**'+'/node_modules/**' }
```

Comment:
I had to add `+` here to avoid `*/` (https://github.com/jsdoc/jsdoc/issues/821, https://github.com/microsoft/tsdoc/issues/166).